semantic_segmentation/scripts/dataset.py

```diff
@@ -14,28 +14,28 @@ def __init__(self, root, transforms, train=True):
         self.imgs = list(sorted(os.listdir(os.path.join(root, "inputs"))))
         count = len(self.imgs)
         if train:
-            self.imgs = self.imgs[int(count*0.8):]
-            self.targets = list(sorted(os.listdir(os.path.join(root,
-                                "semantic_annotations"))))[int(count*0.8):]
+            self.imgs = self.imgs[int(count * 0.8) :]
+            self.targets = list(
+                sorted(os.listdir(os.path.join(root, "semantic_annotations")))
+            )[int(count * 0.8) :]
```

Comment:
warning - in most cases it is a bad idea to use 80:20 this way (better - `train_test_split` with frozen seed, or a list of files)